db/version_set.cc

```diff
@@ -1218,6 +1218,7 @@ void Version::Get(const ReadOptions& read_options, const LookupKey& k,
       sample_file_read_inc(f->file_metadata);
     }
 
+    StopWatchNano timer(env_, true);
```

Comment:
We should only do timing when perf context level allows timing and by level counter is enabled.